src/ui/Deck.js

```diff
@@ -1,11 +1,26 @@
 import Api from "./Api.js";
+import Hand from "./Hand.js";
+import { useState, useCallback } from "react";
 
-class Deck(){
-	constructor(deckCode){
-	
-	 static async newDeck = await().Api.Deck(deckCode);
-	 this.Deck = newDeck
+export default class Deck {
+  constructor(ID) {
+    this.deckID = ID;
+  }
 
-	}
+  async getDeck(id) {
```

Comment:
Doesn't look like you need to send the id in as an argument here. You can just use `this.deckID`.